package.json

```diff
@@ -34,7 +35,9 @@
     "npm": ">=5.x"
   },
   "browserslist": [
-    "> 0.5%, last 2 versions, Firefox ESR, not dead",
+    "> 1%",
```

Comment:
I think that our script `"build": "webpack"` should be updated. As a best practice, the `lib` and  `dist` folders should be removed before running webpack. I suggest using this package (as a dev dependency) https://www.npmjs.com/package/rimraf and update the script to something like: 

`"build": "rimraf lib dist && webpack"`